### DISCUSSION TITLE
List View

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1436] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
The UI of the Data Management > Recipes Data view is really nice to get an overview of recipes. It'd be great if each view throughout Mealie offered a similar layout to quickly browse recipes and see all associated tags/categories in a clean way.

Each layout currently has its pros and cons, and a list view would be a nice additional option to choose from. For example, the compact view shows ratings, but no categories, tags or tools, the large view shows ratings, but no category, and only up to two tags or tools. The Data Management UI shows all categories, tags and tools, but no ratings, and cannot be filtered.

**(Optional): Suggest A Solution**
Each page could have a "View Options" drop-down to select large, compact or list views. When in list view, it would behave very similar to the existing Data Management UI, though also show star rating (pictures probably aren't necessary).

This view could be quite powerful and used for both casual browsing, or management. It should support all columns from Data Management UI, and additionally have ratings and last updated date. Clicking column titles would sort.

Lastly, an "Edit" button would bring up the checkboxes and allow bulk editing right there. This would allow you to do a search, for example, and apply tags only to the results of that search very easily.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by fivestones** [comment]
I think this would be a great addition. Especially as a search result, to easily find a recipe with certain characteristics, and to then also be able to bulk edit recipes in a search result.

**Reply by FeralFlora** [reply]
I agree with the sentiments above. This would be nice, especially on larger screens.

**Comment by hillsandales** [comment]
A compact list view of recipes would be great.

**Comment by pgale** [comment]
Yes particularly if that list view didn't have images and so would help improve load times.

**Comment by gaboriaudn** [comment]
bonjour,
est ce normal que je n'arrive pas à modifier les dates dans la plannification des repas ?
<img width="1648" height="791" alt="image" src="https://github.com/user-attachments/assets/1e1f60a3-8f72-46b5-bc75-6083bd455650" />
